README.rst

```diff
@@ -31,6 +31,16 @@ Installation
 
     pip install -r requirements.txt
 
+Auto-formatting with pre-commit
+-------------------------------
+
+This project is using `pre-commit <https://pre-commit.com/>`_.
+
+#. Install the git pre-commit hooks on your clone::
```

Comment:
It might be confusing to have an enumerated list with only one item. We could have it like in [BugBug](https://github.com/mozilla/bugbug/#auto-formatting) or add another item that requires installing the dependencies in `requirements-test.txt`.